src/Options/ParseOptions.hpp

```diff
@@ -36,6 +36,14 @@
 #include "Utilities/TypeTraits/IsStdArrayOfSize.hpp"
 
 namespace Options {
+/// Option parser tag to retrieve the YAML source.  This tag can be
```

Comment:
Question: would it be reasonable to preserve the source to keep the comments? Or maybe yaml-cpp supports that (less likely)? I'm just thinking of the case where someone might actually care about the comments in the input file. If that's really difficult to do, then we can just ignore it :)